tests/unit/amo/components/TestFooter.js

```diff
@@ -32,19 +19,18 @@ describe('Footer', () => {
     // cause a 404 error.
     // See:
     // github.com/mozilla/addons-frontend/pull/2524#pullrequestreview-42911624
-    expect(root.querySelector('.Footer-privacy-link').textContent).toEqual('Privacy');
-    expect(root.querySelector('.Footer-privacy-link').href).toEqual('https://www.mozilla.org/privacy/websites/');
-    expect(root.querySelector('.Footer-legal-link').textContent).toEqual('Legal');
-    expect(root.querySelector('.Footer-legal-link').href).toEqual('https://www.mozilla.org/about/legal/');
-    expect(root.querySelector('.Footer-cookies-link').textContent).toEqual('Cookies');
-    expect(root.querySelector('.Footer-cookies-link').href).toEqual('https://www.mozilla.org/privacy/websites/#cookies');
-    expect(root.querySelector('.Footer-trademark-abuse-link').textContent).toEqual('Report Trademark Abuse');
-    expect(root.querySelector('.Footer-trademark-abuse-link').href).toEqual('https://www.mozilla.org/about/legal/fraud-report/');
-
+    expect(root.find('.Footer-privacy-link').text()).toEqual('Privacy');
```

Comment:
This works but there's an even nicer helper you can use to write tests like this one:
```js
expect(root.find('.Footer-privacy-link')).toHaveText('Privacy');
```